make-containerized-build.groovy

```diff
@@ -2,13 +2,9 @@ def cico_retries = 16
 def cico_retry_interval = 60
 def ci_git_repo = 'https://github.com/ceph/ceph-csi'
 def ci_git_branch = 'ci/centos'
+def ref = null
```

Comment:
instead of setting to `null`, please set it to the default (`'master'`)

Reply:
Thanks for the review, I'll make the changes.